### PR TITLE
Add TiCDC Debezium Protocol Docs

### PR DESCRIPTION
**Pull request by hongyunyan**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?
https://github.com/pingcap/tiflow/pull/10197
https://github.com/pingcap/tiflow/pull/10307

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/ticdc-debezium.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
+---
+title: TiCDC Debezium Protocol
+summary: 了解 TiCDC Debezium Protocol 的概念和使用方法。
+---
+
+# TiCDC Debezium Protocol
+
+[Debezium](https://debezium.io/) 一种数据库变更数据捕获的工具，它会将捕获的数据库变更的每一条记录转化为一个被称为"事件"的消息，并将这些事件发送到 Kafka 中。 为了便利在数据链路中已经使用 Debezium 的用户，TiCDC 提供了 Debezium 信息格式的协议，用于将 TiDB 的变更以 Debezium 的格式直接传输到 Kafka，来简化用户下游消费的数据链路。
```
Comment (flowbehappy) [inline ticdc/ticdc-debezium.md:8]:
```suggestion
[Debezium](https://debezium.io/) 一种数据库变更数据捕获的工具，它会将捕获的数据库变更的每一条记录转化为一个被称为"事件"的消息，并将这些事件发送到 Kafka 中。 为了便利在数据链路中已经使用 Debezium 的用户，TiCDC 支持将 TiDB 的变更以 Debezium 的格式直接传输到 Kafka，来简化用户下游消费的数据链路。
```

File: ticdc/ticdc-debezium.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+## 使用 Debezium
```
Comment (flowbehappy) [inline ticdc/ticdc-debezium.md:10]:
```suggestion
## 使用 Debezium 消息格式
```

File: ticdc/ticdc-debezium.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+```
+
+以上 JSON 数据的重点字段解释如下：
+
+| 字段      | 类型   | 说明                                                                      |
+|:----------|:-------|:-------------------------------------------------------------------------|
+| payload.op        | String | "c"表示这是一个创建语句，"u"表示这是一个更新语句，"d”表示这是一个删除语句          |
+| payload.ts_ms     | Number | 记录的是 TiCDC 生成这条信息的时间戳（ms 级别）                                |
+| payload.before    | JSON   | 记录的是这条事件语句变更前该语句的值的情况，对于 "c" 事件，before 字段为 null     |
+| payload.after     | JSON   | 记录的是这条事件语句变更后该语句的值的情况，对于 "d" 事件，before 字段为 null     |
```
Comment (flowbehappy) [inline ticdc/ticdc-debezium.md:124]:
```suggestion
| payload.after     | JSON   | 记录的是这条事件语句变更后该语句的值的情况，对于 "d" 事件，after 字段为 null     |
```

File: ticdc/ticdc-debezium.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+
+### 数据类型映射
+
+消息中的数据格式映射基本遵循 https://debezium.io/documentation/reference/2.4/connectors/mysql.html#mysql-data-types 的数据类型映射规则，但是 TiCDC 会对部分数据类型进行特殊处理，具体如下：
+
+1. 目前 TiDB 不支持空间数据类型，包括 GEOMETRY, LINESTRING, POLYGON, MULTIPOINT, MULTILINESTRING, MULTIPOLYGON, GEOMETRYCOLLECTION。
+
+2. 对于 Binary 和 Blob 相关类型， TiCDC 会将其按照 Base64 编码后以 String 类型表示，包括 BINARY, VARBINARY, TINYBLOB, BLOB, MEDIUMBLOB, LONGBLOB。
+
+3. 对于 Enum 和 Set 类型，会使用他们的 String 值来表示。
```
Comment (breezewish) [inline ticdc/ticdc-debezium.md:138]: Debezium 格式好像也是如此

File: ticdc/ticdc-debezium.md [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+| payload.op        | String | "c"表示这是一个 insert ，"u"表示这是一个 update，"d”表示这是一个 delete          |
+| payload.ts_ms     | Number | 记录的是 TiCDC 生成这条信息的时间戳（ms 级别）                                |
+| payload.before    | JSON   | 记录的是这条事件语句变更前该语句的值的情况，对于 "c" 事件，before 字段为 null     |
+| payload.after     | JSON   | 记录的是这条事件语句变更后该语句的值的情况，对于 "d" 事件，after 字段为 null     |
+| payload.source.commit_ts     | Number   | 记录的是 TiCDC 生成这条信息的 CommitTs 标识                    |
+| payload.source.db     | String   | 记录的是对应事件发生的数据库的名称                    |
+| payload.source.table     | String   |  记录的是对应事件发生的数据表的名称                    |
+| schema.fields     | JSON   |  记录了 payload 中各个字段的类型信息，包括对应行数据变更前后 schema 的信息等      |
+
+### 数据类型映射
```
Comment (breezewish) [inline ticdc/ticdc-debezium.md:130]: 作为一个用户，也许我更希望看到 TiCDC 的输出和 Debezium 的输出有何差异（而非单纯的 TiCDC 的输出是什么）

Comment (breezewish) [inline ticdc/ticdc-debezium.md:130]: 是否考虑额外一个 Section 关于 TiCDC 的输出和 Debezium 的输出有何差异？这对于一个已经用了 Debezium + MySQL 要迁移到 TiDB + TiCDC 的用户比较有帮助。这个 Section 可以顺便包含 L22~L24 的内容。

File: ticdc/ticdc-debezium.md [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+
+使用 `Debezium` 时的配置样例如下所示：
+
+```shell
+cdc cli changefeed create --server=http://127.0.0.1:8300 --changefeed-id="kafka-debezium" --sink-uri="kafka://127.0.0.1:9092/topic-name?kafka-version=2.4.0&protocol=debezium"
+```
+
+同时在 Debezium 输出格式中我们包含当前行的 Schema 信息，以便于下游消费者能够更好的理解当前行的数据结构。对于不需要输出 Schema 信息的场景，我们也可以通过在 changefeed 的配置文件或者 `sink-uri` 中将 `debezium-disable-schema` 参数设置为 true 来关闭 Schema 信息的输出。
+
+另外，Debezium 原有格式其中并不包含 TiDB 专有的 CommitTS 事务唯一标识等重要字段。为了保证数据的完整性，TiCDC 在 Debezium 格式中增加了 CommitTs 和 ClusterID 两个字段，用于标识 TiDB 数据变更的相关信息。
```
Comment (lilin90) [inline ticdc/ticdc-debezium.md:24]:
```suggestion
此外，Debezium 原有格式中并不包含 TiDB 专有的 `CommitTS` 事务唯一标识等重要字段。为了保证数据的完整性，TiCDC 在 Debezium 格式中增加了 `CommitTs` 和 `ClusterID` 两个字段，用于标识 TiDB 数据变更的相关信息。
```

File: ticdc/ticdc-debezium.md [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+
+## Message 格式定义
+
+下面将介绍 Debezium 格式输出的 DML Event 的格式定义。
+
+### DML Event
+
+TiCDC 会把一个 DML Event 编码成如下格式:
```
Comment (lilin90) [inline ticdc/ticdc-debezium.md:32]: 一般中文句子里用中文标点，中文文档里的英文字符中间可以用英文标点。
```suggestion
TiCDC 会把一个 DML Event 编码成如下格式：
```

File: ticdc/ticdc-debezium.md [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+| 字段      | 类型   | 说明                                                                      |
+|:----------|:-------|:-------------------------------------------------------------------------|
+| payload.op        | String | "c"表示这是一个 insert ，"u"表示这是一个 update，"d”表示这是一个 delete          |
+| payload.ts_ms     | Number | 记录的是 TiCDC 生成这条信息的时间戳（ms 级别）                                |
+| payload.before    | JSON   | 记录的是这条事件语句变更前该语句的值的情况，对于 "c" 事件，before 字段为 null     |
+| payload.after     | JSON   | 记录的是这条事件语句变更后该语句的值的情况，对于 "d" 事件，after 字段为 null     |
+| payload.source.commit_ts     | Number   | 记录的是 TiCDC 生成这条信息的 CommitTs 标识                    |
+| payload.source.db     | String   | 记录的是对应事件发生的数据库的名称                    |
+| payload.source.table     | String   |  记录的是对应事件发生的数据表的名称                    |
+| schema.fields     | JSON   |  记录了 payload 中各个字段的类型信息，包括对应行数据变更前后 schema 的信息等      |
```
Comment (lilin90) [inline ticdc/ticdc-debezium.md:128]:
```suggestion
| payload.op        | 字符串 | `"c"` 表示这是一个 insert，`"u"` 表示这是一个 update，`"d"` 表示这是一个 delete  |
| payload.ts_ms     | 数值 | 记录的是 TiCDC 生成这条信息的时间戳（毫秒级别）                                |
| payload.before    | JSON   | 记录的是这条事件语句变更前该语句的值的情况，对于 `"c"` 事件，`before` 字段的值为 `null`     |
| payload.after     | JSON   | 记录的是这条事件语句变更后该语句的值的情况，对于 `"d"` 事件，`after` 字段的值为 `null`     |
| payload.source.commit_ts     | 数值  | 记录的是 TiCDC 生成这条信息的 `CommitTs` 标识                    |
| payload.source.db     | 字符串   | 记录的是对应事件发生的数据库的名称                    |
| payload.source.table     | 字符串  |  记录的是对应事件发生的数据表的名称                    |
| schema.fields     | JSON   |  记录了 payload 中各个字段的类型信息，包括对应行数据变更前后 schema 的信息等      |
```

File: ticdc/ticdc-debezium.md [hunk 9]
```diff
@@ -0,0 +1,138 @@
...
+
+### 数据类型映射
+
+消息中的数据格式映射基本遵循 https://debezium.io/documentation/reference/2.4/connectors/mysql.html#mysql-data-types 的数据类型映射规则，大体与 Debezium Connector 原生的 Message 保持一致。但是对于部分数据类型，TiCDC Debezium 的处理方式与 Debezium Connector Message 存在一定差异，具体如下：
```
Comment (lilin90) [inline ticdc/ticdc-debezium.md:132]:
```suggestion
TiCDC Debezium 消息中的数据格式映射基本遵循 [Debezium 的数据类型映射规则](https://debezium.io/documentation/reference/2.4/connectors/mysql.html#mysql-data-types)，大体与 Debezium Connector 原生的 Message 保持一致。但是对于部分数据类型，TiCDC Debezium 的处理方式与 Debezium Connector Message 存在一定差异，具体如下：
```

File: ticdc/ticdc-debezium.md [hunk 10]
```diff
@@ -0,0 +1,138 @@
...
+
+1. 目前 TiDB 不支持空间数据类型，包括 GEOMETRY, LINESTRING, POLYGON, MULTIPOINT, MULTILINESTRING, MULTIPOLYGON, GEOMETRYCOLLECTION。
+
+2. 对于 String-likes 的数据类型，包括 Varchar, String, VarString, TinyBlob, MediumBlob, BLOB, LongBlob 等, 当该列具有 BINARY 标志是，TiCDC 会将其按照 Base64 编码后以 String 类型表示。如果没有则直接编码为 String 类型。 而原生的 Debezium Connector 会根据 binary.handling.mode 以不同的编码方式进行编码。
```
Comment (lilin90) [inline ticdc/ticdc-debezium.md:136]:
```suggestion
2. 对于 String-likes 的数据类型，包括 Varchar, String, VarString, TinyBlob, MediumBlob, BLOB, LongBlob 等，当该列具有 BINARY 标志时，TiCDC 会将其按照 Base64 编码后以 String 类型表示；当该列没有 BINARY 标志时，TiCDC 则直接将其编码为 String 类型。而原生的 Debezium Connector 会根据 `binary.handling.mode` 以不同的编码方式进行编码。
```

File: ticdc/ticdc-debezium.md [hunk 11]
```diff
@@ -0,0 +1,138 @@
...
+
+3. 对于 Decimal 数据类型，TiCDC 均会使用 float64 类型来表示，包括 DECIMAL, NUMERIC。对于原生的 Debezium Connector，根据数据类型的不同精度会采用 float32 或者 float 64 的方式进行编码。
```
Comment (lilin90) [inline ticdc/ticdc-debezium.md:138]:
```suggestion
3. 对于 Decimal 数据类型，TiCDC 均会使用 float64 类型来表示，包括 `DECIMAL` 和 `NUMERIC`。原生的 Debezium Connector 会根据数据类型的不同精度采用 float32 或者 float64 的方式进行编码。
```

File: ticdc/ticdc-debezium.md [hunk 12]
```diff
@@ -0,0 +1,138 @@
...
+summary: 了解 TiCDC Debezium Protocol 的概念和使用方法。
+---
+
+# TiCDC Debezium Protocol
+
+[Debezium](https://debezium.io/) 是一个用于捕获数据库变更的工具。它会将捕获的数据库变更的每一条记录转化为一个被称为“事件”(event) 的消息，并将这些事件发送到 Kafka 中。为了便利在数据链路中已经使用 Debezium 的用户，TiCDC 支持将 TiDB 的变更以 Debezium 的格式直接传输到 Kafka，来简化用户下游消费的数据链路。
+
+## 使用 Debezium 消息格式
+
+当使用 Kafka 作为下游 Sink 的时候，你可以在 `sink-uri` 中指定使用 Debezium，TiCDC 将以 Event 为基本单位封装构造 Debezium Message，向下游发送 TiDB 的数据变更事件。
```
Comment (CharlesCheung96) [inline ticdc/ticdc-debezium.md:12]:
```suggestion
当使用 Kafka 作为下游 Sink 的时候，你可以在 `sink-uri` 的 protocol字段中指定使用 Debezium，TiCDC 将以 Event 为基本单位封装构造 Debezium Message，向下游发送 TiDB 的数据变更事件。
```

**Review by flowbehappy — APPROVED** [review]

**Review by lilin90 — COMMENTED** [review]
/lgtm

**Review by lilin90 — APPROVED** [review]

File: ticdc/ticdc-debezium.md [hunk 13]
```diff
@@ -0,0 +1,138 @@
...
+
+# TiCDC Debezium Protocol
+
+[Debezium](https://debezium.io/) 是一个用于捕获数据库变更的工具。它会将捕获的数据库变更的每一条记录转化为一个被称为“事件”(event) 的消息，并将这些事件发送到 Kafka 中。为了便利在数据链路中已经使用 Debezium 的用户，TiCDC 支持将 TiDB 的变更以 Debezium 的格式直接传输到 Kafka，来简化用户下游消费的数据链路。
+
+## 使用 Debezium 消息格式
+
+当使用 Kafka 作为下游 Sink 的时候，你可以在 `sink-uri` 的 protocol 字段中指定使用 Debezium，TiCDC 将以 Event 为基本单位封装构造 Debezium Message，向下游发送 TiDB 的数据变更事件。
+
+TiCDC Debezium Protocol 以 Event 为基本单位向下游复制数据变更事件。目前，Debezium 协议只支持 Row Changed Event，会直接忽略 DDL Event 和 WATERMARK Event。Row Changed Event 代表一行的数据变化，在行发生变更时该 Event 被发出，包含该行在变更前后的相关信息。
```
Comment (qiancai) [inline ticdc/ticdc-debezium.md:14]:
```suggestion
当使用 Kafka 作为下游 Sink 时，你可以将 `sink-uri` 的 `protocol` 字段指定为 `debezium`，TiCDC 将以 Event 为基本单位封装构造 Debezium 消息，向下游发送 TiDB 的数据变更事件。

目前，Debezium 协议只支持 Row Changed Event，会直接忽略 DDL Event 和 WATERMARK Event。Row Changed Event 代表一行的数据变化，在行发生变更时该 Event 被发出，包含该行在变更前后的相关信息。
```

File: ticdc/ticdc-debezium.md [hunk 14]
```diff
@@ -0,0 +1,138 @@
...
+        }
+    }
+}
+```
+
+以上 JSON 数据的重点字段解释如下：
+
+| 字段      | 类型   | 说明                                                                      |
+|:----------|:-------|:-------------------------------------------------------------------------|
+| payload.op        | 字符串 | `"c"` 表示这是一个 insert，`"u"` 表示这是一个 update，`"d"` 表示这是一个 delete  |
```
Comment (qiancai) [inline ticdc/ticdc-debezium.md:121]:
```suggestion
| payload.op        | 字符串 | 变更事件类型。`"c"` 表示这是一个 `INSERT` 事件，`"u"` 表示这是一个 `UPDATE` 事件，`"d"` 表示这是一个 `DELETE` 事件  |
```

File: ticdc/ticdc-debezium.md [hunk 15]
```diff
@@ -0,0 +1,138 @@
...
+
+### 数据类型映射
+
+TiCDC Debezium 消息中的数据格式映射基本遵循 [Debezium 的数据类型映射规则](https://debezium.io/documentation/reference/2.4/connectors/mysql.html#mysql-data-types)，大体与 Debezium Connector 原生的 Message 保持一致。但是对于部分数据类型，TiCDC Debezium 的处理方式与 Debezium Connector Message 存在一定差异，具体如下：
+
+1. 目前 TiDB 不支持空间数据类型，包括 GEOMETRY、LINESTRING、POLYGON、MULTIPOINT、MULTILINESTRING、MULTIPOLYGON、GEOMETRYCOLLECTION。
+
+2. 对于 String-likes 的数据类型，包括 Varchar、String、VarString、TinyBlob、MediumBlob、BLOB、LongBlob 等，当该列具有 BINARY 标志时，TiCDC 会将其按照 Base64 编码后以 String 类型表示；当该列没有 BINARY 标志时，TiCDC 则直接将其编码为 String 类型。而原生的 Debezium Connector 会根据 `binary.handling.mode` 以不同的编码方式进行编码。
+
+3. 对于 Decimal 数据类型，TiCDC 均会使用 float64 类型来表示，包括 `DECIMAL` 和 `NUMERIC`。原生的 Debezium Connector 会根据数据类型的不同精度采用 float32 或者 float64 的方式进行编码。
```
Comment (qiancai) [inline ticdc/ticdc-debezium.md:138]:
```suggestion
- 目前 TiDB 不支持空间数据类型，包括 GEOMETRY、LINESTRING、POLYGON、MULTIPOINT、MULTILINESTRING、MULTIPOLYGON、GEOMETRYCOLLECTION。

- 对于 String-likes 的数据类型，包括 Varchar、String、VarString、TinyBlob、MediumBlob、BLOB、LongBlob 等，当该列具有 BINARY 标志时，TiCDC 会将其按照 Base64 编码后以 String 类型表示；当该列没有 BINARY 标志时，TiCDC 则直接将其编码为 String 类型。而原生的 Debezium Connector 会根据 `binary.handling.mode` 以不同的编码方式进行编码。

- 对于 Decimal 数据类型，包括 `DECIMAL` 和 `NUMERIC`，TiCDC 均会使用 float64 类型来表示。而原生的 Debezium Connector 会根据数据类型的不同精度采用 float32 或者 float64 的方式进行编码。
```

File: ticdc/ticdc-debezium.md [hunk 16]
```diff
@@ -0,0 +1,138 @@
...
+# TiCDC Debezium Protocol
+
+[Debezium](https://debezium.io/) 是一个用于捕获数据库变更的工具。它会将捕获的数据库变更的每一条记录转化为一个被称为“事件”(event) 的消息，并将这些事件发送到 Kafka 中。为了便利在数据链路中已经使用 Debezium 的用户，TiCDC 支持将 TiDB 的变更以 Debezium 的格式直接传输到 Kafka，来简化用户下游消费的数据链路。
+
+## 使用 Debezium 消息格式
+
+当使用 Kafka 作为下游 Sink 的时候，你可以在 `sink-uri` 的 protocol 字段中指定使用 Debezium，TiCDC 将以 Event 为基本单位封装构造 Debezium Message，向下游发送 TiDB 的数据变更事件。
+
+TiCDC Debezium Protocol 以 Event 为基本单位向下游复制数据变更事件。目前，Debezium 协议只支持 Row Changed Event，会直接忽略 DDL Event 和 WATERMARK Event。Row Changed Event 代表一行的数据变化，在行发生变更时该 Event 被发出，包含该行在变更前后的相关信息。
+
```
Comment (qiancai) [inline ticdc/ticdc-debezium.md:15]: 这里要不要也解释下什么是 WATERMARK Event

File: ticdc/ticdc-debezium.md
Comment (qiancai) [inline ticdc/ticdc-debezium.md:1]: 这两处要加下 Debezium 格式吗

1. https://docs.pingcap.com/zh/tidb/dev/ticdc-overview#ticdc-%E4%B8%BB%E8%A6%81%E7%89%B9%E6%80%A7

image

2. https://docs.pingcap.com/zh/tidb/dev/ticdc-changefeed-config#ticdc-changefeed-%E9%85%8D%E7%BD%AE%E6%96%87%E4%BB%B6%E8%AF%B4%E6%98%8E

image

Comment (hongyunyan) [reply ticdc/ticdc-debezium.md:1]: 嗯嗯应该也是需要的

Comment (lilin90) [reply ticdc/ticdc-debezium.md:1]: 如果需要，@hongyunyan 在这个 PR 里也更新下相关的两个文档？

File: ticdc/ticdc-debezium.md [hunk 17]
```diff
@@ -0,0 +1,138 @@
+---
+title: TiCDC Debezium Protocol
+summary: 了解 TiCDC Debezium Protocol 的概念和使用方法。
+---
+
+# TiCDC Debezium Protocol
+
+[Debezium](https://debezium.io/) 是一个用于捕获数据库变更的工具。它会将捕获的数据库变更的每一条记录转化为一个被称为“事件” (event) 的消息，并将这些事件发送到 Kafka 中。从 v8.0.0 起，TiCDC 支持将 TiDB 的变更以 Debezium 的格式直接传输到 Kafka，简化了使用 Debezium 的数据链路。
```
Comment (hongyunyan) [inline ticdc/ticdc-debezium.md:8]:
```suggestion
[Debezium](https://debezium.io/) 是一个用于捕获数据库变更的工具。它会将捕获的数据库变更的每一条记录转化为一个被称为“事件” (event) 的消息，并将这些事件发送到 Kafka 中。从 v8.0.0 起，TiCDC 支持将 TiDB 的变更以 Debezium 的格式直接传输到 Kafka，为之前使用 Debezium 的 MySQL 集成的用户简化了从 MySQL 数据库的迁移的流程。
```

File: ticdc/ticdc-debezium.md [hunk 18]
```diff
@@ -0,0 +1,138 @@
...
+| payload.before    | JSON   | 这条事件语句变更前的数据值，对于 `"c"` 事件，`before` 字段的值为 `null`     |
+| payload.after     | JSON   | 这条事件语句变更后的数据值，对于 `"d"` 事件，`after` 字段的值为 `null`     |
+| payload.source.commit_ts     | 数值  | TiCDC 生成这条信息时的 `CommitTs` 标识                    |
+| payload.source.db     | 字符串   | 事件发生的数据库的名称                    |
+| payload.source.table     | 字符串  |  事件发生的数据表的名称                    |
+| schema.fields     | JSON   |  payload 中各个字段的类型信息，包括对应行数据变更前后 schema 的信息等      |
+
+### 数据类型映射
+
+TiCDC Debezium 消息中的数据格式映射基本遵循 [Debezium 的数据类型映射规则](https://debezium.io/documentation/reference/2.4/connectors/mysql.html#mysql-data-types)，大体与 Debezium Connector 原生的 Message 保持一致。但是对于部分数据类型，TiCDC Debezium 的处理方式与 Debezium Connector Message 存在一定差异，具体如下：
```
Comment (hongyunyan) [inline ticdc/ticdc-debezium.md:132]:
```suggestion
TiCDC Debezium 消息中的数据格式映射基本遵循 [Debezium 的数据类型映射规则](https://debezium.io/documentation/reference/2.4/connectors/mysql.html#mysql-data-types)，与 Debezium Connector for MySQL 原生消息大体一致。但是对于部分数据类型，TiCDC Debezium 的处理方式与 Debezium Connector Message 存在一定差异，具体如下：
```

File: ticdc/ticdc-debezium.md [hunk 19]
```diff
@@ -0,0 +1,138 @@
...
+
+# TiCDC Debezium Protocol
+
+[Debezium](https://debezium.io/) 是一个用于捕获数据库变更的工具。它会将捕获的数据库变更的每一条记录转化为一个被称为“事件” (event) 的消息，并将这些事件发送到 Kafka 中。从 v8.0.0 起，TiCDC 支持将 TiDB 的变更以 Debezium 的格式直接传输到 Kafka，简化了使用 Debezium 的数据链路。
+
+## 使用 Debezium 消息格式
+
+当使用 Kafka 作为下游 Sink 时，你可以将 `sink-uri` 的 `protocol` 字段指定为 `debezium`，TiCDC 将以 Event 为基本单位封装构造 Debezium 消息，向下游发送 TiDB 的数据变更事件。
+
+目前，Debezium 协议只支持 Row Changed Event，会直接忽略 DDL Event 和 WATERMARK Event（WATERMARK Event 一般用于标记一个表的同步进度，表示所有小于 watermark 的事件都已经发送给下游。）。Row Changed Event 代表一行的数据变化，在行发生变更时该 Event 被发出，包含该行在变更前后的相关信息。
```
Comment (lilin90) [inline ticdc/ticdc-debezium.md:14]:
```suggestion
目前，Debezium 协议只支持 Row Changed Event，会直接忽略 DDL Event 和 WATERMARK Event。Row Changed Event 代表一行的数据变化，在行发生变更时该 Event 被发出，包含该行在变更前后的相关信息。WATERMARK Event 用于标记一个表的同步进度，所有小于 watermark 的事件都已经发送给下游。
```

File: ticdc/ticdc-overview.md [hunk 20]
```diff
@@ -23,7 +23,7 @@ TiCDC 提供了以下核心能力：
 - 提供 TiDB -> TiDB 之间数据容灾复制的能力，实现秒级别 RPO 和分钟级别 RTO
 - 提供 TiDB 之间双向复制的能力，支持通过 TiCDC 构建多写多活的 TiDB 集群
 - 提供 TiDB -> MySQL（或其他兼容 MySQL 协议的数据库）的低延迟的增量数据同步能力
-- 提供 TiDB -> Kafka 增量数据同步能力，推荐的数据格式包含 [Canal-JSON](/ticdc/ticdc-canal-json.md)，[Avro](/ticdc/ticdc-avro-protocol.md) 等
+- 提供 TiDB -> Kafka 增量数据同步能力，推荐的数据格式包含 [Canal-JSON](/ticdc/ticdc-canal-json.md)，[Avro](/ticdc/ticdc-avro-protocol.md)，[Debezium](/ticdc/ticdc-debezium.md) 等
 - 提供 TiDB -> 存储服务（如：Amazon S3、GCS、Azure Blob Storage 和 NFS）增量数据同步能力。
```
Comment (lilin90) [inline ticdc/ticdc-overview.md:27]:
```suggestion
- 提供 TiDB -> 存储服务（如：Amazon S3、GCS、Azure Blob Storage 和 NFS）增量数据同步能力
```

**Comment by lilin90** [comment]
@qiancai All your comments have been addressed. PTAL again~ Thanks!

**Review by qiancai — APPROVED** [review]

File: ticdc/ticdc-debezium.md [hunk 21]
```diff
@@ -0,0 +1,138 @@
...
+
+# TiCDC Debezium Protocol
+
+[Debezium](https://debezium.io/) 是一个用于捕获数据库变更的工具。它会将捕获的数据库变更的每一条记录转化为一个被称为“事件” (event) 的消息，并将这些事件发送到 Kafka 中。从 v8.0.0 起，TiCDC 支持将 TiDB 的变更以 Debezium 的格式直接传输到 Kafka，为之前使用 Debezium 的 MySQL 集成的用户简化了从 MySQL 数据库迁移的过程。
+
+## 使用 Debezium 消息格式
+
+当使用 Kafka 作为下游 Sink 时，你可以将 `sink-uri` 的 `protocol` 字段指定为 `debezium`，TiCDC 将以 Event 为基本单位封装构造 Debezium 消息，向下游发送 TiDB 的数据变更事件。
+
+目前，Debezium 协议只支持 Row Changed Event，会直接忽略 DDL Event 和 WATERMARK Event。Row Changed Event 代表一行的数据变化，在行发生变更时该 Event 被发出，包含该行在变更前后的相关信息。WATERMARK Event 用于标记一个表的同步进度，所有小于 watermark 的事件都已经发送给下游。
```
Comment (lilin90) [inline ticdc/ticdc-debezium.md:14]:
```suggestion
目前，Debezium 协议只支持 Row Changed Event，会直接忽略 DDL Event 和 WATERMARK Event。Row Changed Event 代表一行的数据变化，在行发生变更时该 Event 被发出，包含该行在变更前后的相关信息。WATERMARK Event 用于标记一个表的同步进度，所有早于 watermark 的事件都已经发送给下游。
```

**Pull request merged**